app/admin/facility/components/dark-mode-toggle.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
```

Comment:
Can you delete this component along with all the other darkModeToggle components in the other places as I don't think they are used anywhere